src/containerConfig.ts

```diff
@@ -49,8 +50,8 @@ export const registerExternalValues = (options?: RegisterOptions): DependencyCon
     { token: SERVICES.METER, provider: { useValue: OtelMetrics.getMeterProvider().getMeter(SERVICE_NAME) } },
     { token: SERVICES.QUEUE_CLIENT, provider: { useFactory: instancePerContainerCachingFactory(queueClientFactory) } },
     { token: HANDLERS.NEW, provider: { useClass: NewJobHandler } },
-    { token: HANDLERS.UPDATE, provider: { useClass: NewJobHandler } },
-    { token: HANDLERS.SWAP, provider: { useClass: NewJobHandler } },
+    { token: HANDLERS.UPDATE, provider: { useClass: UpdateJobHandler } },
+    { token: HANDLERS.SWAP, provider: { useClass: UpdateJobHandler } },
```

Comment:
Can you avoid duplicate and check with || 'or' :
`{ token: HANDLERS.UPDATE || HANDLERS.SWAP, provider: { useClass: UpdateJobHandler } },`